Add an explanation for certification (#371)
opengisch/QField-docs: documentation/get-started/certification.en.md

---
title: Certification
tx_slug: documentation_get-started_certifiation
---

# Certification

The QField Certificate Program is designed to foster community involvement in the QField project and provide high-quality education for the QField application. Therefore, the application process requires applicants to both detail their contributions to the QField project and make their training materials available for review. Activities that contribute to the QField project include:

- Contributions to the QField documentation and training materials

- Translation of QField materials

- Filing bug reports

- Financially supporting new feature development

> - Financial contributions to and sponsorship of QField

Reviewer: Can/should these two be unified?

- Involvement in your local QField User Group

> - Publication of open QField course ware

Reviewer 2: ```suggestion
- Publication of open QField courseware
```

To contact the certification team, please write to info@opengis.ch.

## Certification Process

> Each of your courses requires you to provide training material. This includes exercises, lectures, and data. This material will be reviewed to ensure that it is current, accurate, and of high quality. If the material is not of adequate quality, it may be refused.

Reviewer: ```suggestion
Each of your courses requires a check of the training material. This includes exercises, lectures, and data. This material will be reviewed to ensure that it is current, accurate, and of high quality. If the material is not of adequate quality, it may be refused.
```

The review will be handled by OPENGIS.ch.

> If you are accepted as a QField Certified Organization, your contributions will be published for transparency.

Author: As a certified trainer

> If approved, you are required to pay a €20 certification fee to the QField project for each certificate you issue. This establishes a mechanism for financial support for the project. Payments for certificates are made either using online credit card transactions or, on request, by regular invoice and SEPA wire transfer. Payments can be handled in batches for many certificates at once.

Reviewer: ```suggestion
As an approved trainer, you will be able to hand out official QField certificates for course participants. For each certificate, a €20 certification fee is paid to QField, this establishes a mechanism for financial support for the project. Payments for certificates are made either using credit card transactions or, on request, by regular invoice and SEPA wire transfer.
```

I wonder if we should also put the technical details for payment handling "credit card, wire transfer, SEPA, ..." somewhere else, I think it doesn't really belong into the scope here.

> You can apply through this [form](https://changelog.qgis.org/en/qfield/create-certifyingorganisation/).<!-- markdown-link-check-disable-line -->

Reviewer: ```suggestion
Start your certificatoin journey by [applying as a certification organisation](https://changelog.qgis.org/en/qfield/create-certifyingorganisation/) now.<!-- markdown-link-check-disable-line -->
```
